Second Disinflation Act of Azuria - Nuking the Keep Vault

WeNeedMorePhoron opened the pull request:
## About The Pull Request
Experimental economic PR to reduce the amount of mammons the keep get while making it more **dynamic**
- Vault mechanic is removed. Mapping is NOT touched because fuck that fuck that fuck that.
- Instead of transporting into the vault, the stockpile now have a mechanic called **Minting**. Minting is when a valuable item (see criteria) is left clicked on the stockpile. They get instantly deleted and 80% of its sell price will be given to the treasury. Then, a percentage based on the collectable treasure percentage is given to the seller's account
-  Criteria: Any valuables that is not clothing or armor above 30 value, that is not weapons, ores or clothing. statues, cups, rings, platters, and candlesticks are always accepted regardless of value.).
- Bathmatron's Vault is now at 15%.
- Treasury starting amount now randomize between 1000 - 2000 instead of 500 - 1000.
- To give the Keep / lowpop **some** safety pool, there's a passive generation of +50 mammon every treasury tick, which is unified to every 6 minutes. This add up to around +1500 or so mammons over an average round and is not enough to satisfy any real economic need on highpop if the Keep is being spendy.
- Stockpile limits of a lot of niche food good has been raised by 10 to reduce micromanagement because steward and soilson cooperation is now a big source of income for the duchy.

<!-- Describe your pull request. Avoid text walls, use concise bullet points for easier readability. Document every change, or this can delay review and even discourage maintainers from merging your PR. -->

## Testing Evidence
Tested

<!-- It's mandatory to test your PR. Provide images, clips or description of how you tested your changes where possible. -->

## Why It's Good For The Game
Attempt to reduce the amount of free money the Keep can get by stockpiling things while making the economy more dynamic by making the Keep's income depends **directly** on how much they can squeeze out of economic activities (I.e. export and people selling treasure and them keeping the differences) instead of hoarding 20 treasures from the Keep at roundstart and then printing out infinite money for the rest of the round.

Need to be TMed to see its actual effects. 

<!-- Argue for the merits of your changes and how they benefit the game, especially if they are controversial. If you can't, then it probably isn't good for the game in the first place. -->

AllCorruptingWyrm commented:
I like this but I think we need to stop making people play adventurer before we try to do this further, perhaps?

AisuDracov commented:
> * Vault mechanic is removed. Mapping is NOT touched because fuck that fuck that fuck that.
> * Instead of transporting into the vault, the stockpile now have a mechanic called **Minting**. Minting is when a valuable item (see criteria) is left clicked on the stockpile. They get instantly deleted and 80% of its sell price will be given to the treasury. Then, a percentage based on the collectable treasure percentage is given to the seller's account

If I'm reading this right, the pay out to the seller is separate from what the keep gets? Say that a treasure is worth 100 and the treasure bounty is set to 70. Does that mean the Keep gets 80 while the seller gets 70? How does that work out when the item's only worth 100, yet it somehow just generated 150 in total? Or does the Keep get 80 and then the 70 is taken out from that, making the Keep earn a net total of 10?

If the latter's the case, then yeah, the steward will rarely ever set up treasure bounty to be as high as 80%, meaning the merchant and bathhouse will become the main ways to sell treasures. Plus, if there's no means to add just pure value to the Keep with the vault mechanic gone, then its gonna be hard to grow it without the steward somehow accumulating a lot of money trying to do so. An example would be how a good steward would have the gems turned into rings to increase the value, but the steward wouldn't sell the treasures to the keep, they'd just shove them into the vault. But now if the vault mechanic is removed, the steward would have to sell the rings to add their value to the keep?

Just not sure how the steward's expected to grow the vault without having to rely on people selling treasures.

AllCorruptingWyrm commented:
Thinking on it; It sounds like a good, reliable way for this to happen is to start seizing and shaking people down for money which I love as it drives conflict but on the other hand ppl are gonna lose their shit, and yes frok I am aware.

MortoSasye commented:
This will turn steward into an active role that has to constantly invest into acquiring treasures to sell and keep the economy afloat, and possibly crash it when no one is handling the vault. Not a big fan.

AisuDracov commented:
Ah, okay. That all makes more sense more. And I forget that the steward can export things too to earn money for the keep. Been meaning to play a steward character for some time, but haven't gotten around to it, so I miss out/forget about all the things they can do.

chemistrymain3 commented:
> Interest is fun. It allows for fun things to happen via money, for the steward and merchant. This current set up, as it reads, requires people to do a chore for their income; doing a rote towner task to generate items, or to drag back loot items from the same dungeon for the 30th time. Given both lose out on popularity in general, I'm not sure this is wise.
> 
> Realistically, imo, the only stockpile items that are going to be meaningful both in volume and price is food exporting, which'll require an active soiler. Which there isn't. Adventurers already chafe at the thought of selling treasurers to the keep at the 20% bounty value when they can just get most of it from the merchant, too.

None of this is true.

WeNeedMorePhoron commented:
Updated the PR based on what I've observed over the past day. 

+10 Stockpile Limit for some food. 
And added in "Rural Tax" which is a passive generation of +50 Mammons every treasury tick (now every 6 minutes instead of between 5 to 8, so a slight boost to export mechanics since demand recover much faster now).
